### ISSUE TITLE
Update step `4-projects` to use workspace module as build process

### ISSUE THREAD

**Issue by felipecrescencio-cit** [terraform-google-modules/terraform-example-foundation#812]
### TL;DR

Update step `4-projects` to use the [TF Cloud Build Workspace](https://github.com/terraform-google-modules/terraform-google-bootstrap/tree/master/modules/tf_cloudbuild_workspace) module for the creation of the [App Infra Pipeline](https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/docs/GLOSSARY.md#app-infra-pipeline).

The Terraform Cloud Build Builder/Runner image created in step `0-bootstrap` should be used instead of creating a new Cloud Build Builder/Runner image.
Using the same image in the Foundation Pipeline and in the App Infra Pipeline will help simplify maintenance, like keeping the versions of the binaries used by the `tf-wrapper.sh` script consistent.
